helios-testing/src/main/java/com/spotify/helios/testing/HeliosSoloDeployment.java

```diff
@@ -74,18 +85,19 @@
  * master and one Helios agent deployed in Docker. Helios Solo uses the Docker instance it is
  * deployed on to run its jobs.
  */
-public class HeliosSoloDeployment implements HeliosDeployment {
+class HeliosSoloDeployment implements HeliosDeployment {
```

Comment:
changing the access modifier will force any users of the previous style of HeliosSoloDeployment/TemporaryJobs fail to compile - is that intentional?